src/main/scala/xiangshan/frontend/IFU.scala

```diff
@@ -821,8 +823,11 @@ class NewIFU(implicit p: Parameters) extends XSModule
     io.toIbuffer.bits.pd(0).isCall  := isCall
     io.toIbuffer.bits.pd(0).isRet   := isRet
 
-    io.toIbuffer.bits.acf(0) := mmio_resend_af
-    io.toIbuffer.bits.ipf(0) := mmio_resend_pf
+    when (mmio_resend_pf) {
```

Comment:
typo?

Reply:
Also, why `mmio_resend_gpf` is not needed here?

Reply:
Thanks for pointing out.

Reply:
Amazing, so MMIO + Guest OS is broken before?

Reply:
@cebarobot

Reply:
It seems quite abnormal for guest OS to fetch instructions from MMIO space. I guess that's why this hasn't caused any problems yet.

Reply:
Seems that we will address this proplem in another PR. Let me resolve this conversation.